packages/server/lib/automation/commands/key_press.ts

```diff
@@ -0,0 +1,85 @@
+import type { KeyPressParams, KeyPressSupportedKeys } from '@packages/types'
+import type { SendDebuggerCommand } from '../../browsers/cdp_automation'
+import type { Client } from 'webdriver'
+
+import Debug from 'debug'
+
+const debug = Debug('cypress:server:automation:command:keypress')
+
+interface KeyCodeLookup extends Record<KeyPressSupportedKeys, string> {}
+
+const invalidKeyErrorKind = 'InvalidKeyError'
+
+export class InvalidKeyError extends Error {
+  kind = invalidKeyErrorKind
+  constructor (key: string) {
+    super(`${key} is not supported by 'cy.press()'.`)
+  }
+  static isInvalidKeyError (e: any): e is InvalidKeyError {
+    return e.kind === invalidKeyErrorKind
+  }
+}
+
+export const CDP_KEYCODE: KeyCodeLookup = {
+  'Tab': 'U+000009',
+}
+
+export async function cdpKeyPress ({ key }: KeyPressParams, send: SendDebuggerCommand): Promise<void> {
+  debug('cdp keypress', { key })
+  if (!CDP_KEYCODE[key]) {
+    throw new InvalidKeyError(key)
+  }
+
+  const keyIdentifier = CDP_KEYCODE[key]
+
+  try {
+    await send('Input.dispatchKeyEvent', {
+      type: 'rawKeyDown',
```

Comment:
What's the difference between `rawKeyDown` and `keyDown`? Could be worth a comment to explain.